cfme/v2v/migration_plans.py

```diff
@@ -442,6 +444,17 @@ def get_plan_vm_list(self):
         request_details_list = view.migration_request_details_list
         return request_details_list
 
+    def wait_for_state(self, state):
+        try:
+            if state in self.MIGRATION_STATES.keys():
```

Comment:
Optional: Could do `method = self.MIGRATION_STATES.get(state)`, then have a check against whether `method` is `None` and raise a ValueError. Something like, 

```python,
method = self.MIGRATION_STATES.get(state)
if method:
    return bool(method(self))
else:
    raise ValueError(<msg>)
```